src/Elmish.WPF/Merge.fs

```diff
@@ -40,9 +40,9 @@ module CollectionTarget =
       Move = oc.Move
       Clear = oc.Clear
       Enumerate = fun () -> upcast oc
-      BoxedCollection = fun () -> oc |> box }
+      GetCollection = fun () -> oc }
 
-  let map (fOut: 'a -> 'b) (fIn: 'b -> 'a) (ct: CollectionTarget<'a>) : CollectionTarget<'b> =
+  let mapA (fOut: 'a0 -> 'a1) (fIn: 'a1 -> 'a0) (ct: CollectionTarget<'a0, 'aCollection>) : CollectionTarget<'a1, 'aCollection> =
```

Comment:
This can be private, right?

Reply:
Josh says that these can be made private, though the next PR will undo that.